locale/pt_PT.utf8/LC_MESSAGES/rosariosis.po

```diff
@@ -163,13 +163,13 @@ msgstr "OK"
 
 #: index.php:86
 msgid "Your account has not yet been activated."
-msgstr ""
+msgstr "Sua conta ainda não foi ativada."
 
 #: index.php:86
 #: index.php:151
 #, fuzzy
 msgid "You will be notified when it has been verified by a school administrator."
-msgstr "Sua conta foi criada. Você será notificado por e-mail quando é verificada pela administração da escola e você pode log in"
+msgstr "Sua conta foi criada. Você será notificado por e-mail e quando for verificado pela administração da escola você pode efetuar o login"
```

Comment:
Here, the `fuzzy` mention is still present, though you have translated / updated the translation.